src/ui/actions/timeline.ts

```diff
@@ -523,19 +532,19 @@ export function setFocusRegion(
       // probably we should also run "getPointNearTime", store that, and take the
       // closest thing we can find of the points we have (assuming we don't have
       // an exact match.)
-      const startIndex = sortedIndexBy(
+      const startIndex = sortedLastIndexBy(
```

Comment:
`sortedLastIndex - 1` is equivalent to saying "the very last index we could use while still containing the entire asked-for region.